app/services/notifications/setup/fetch-recipients.service.js

```diff
@@ -144,14 +146,16 @@ const { db } = require('../../../../db/db.js')
  * ]
  * ```
  *
- * @param {string} dueDate - The 'due' date for outstanding return logs to fetch contacts for
- * @param {string} summer - Whether we are looking for outstanding summer or all year return logs
- * @param {string[]} removeLicences - The licences to exclude from the results
+ * @param {module:SessionModel} session - The session instance to format
  *
  * @returns {Promise<object[]>} The contact data for all the outstanding return logs
  */
-async function go(dueDate, summer, removeLicences) {
-  const { rows } = await _fetch(dueDate, summer, removeLicences)
+async function go(session) {
+  const { returnsPeriod, summer } = DetermineReturnsPeriodService.go(session.returnsPeriod)
```

Comment:
In the next PR we can introduce the journey context and move this into something like `_fetchMultpleLicences()` and move the js docs into this private function in preparation for the `_fetchByLicences()` / 'ad-hoc' step to be introduced. 

Just felt alot was changing that it would be better to separate them out.